docs/operator-manual/airgap/02-install.md

```diff
@@ -10,111 +10,79 @@ you will need a private OCI registry accessible by your Kubernetes cluster. Kube
+```bash
+airgap list | tee artifacts.json
 ```
 
-Download `cert-manager` if it is not installed in the air gap cluster.
+List is assembled from kubewarden helm repository, image and policy lists in [release page](https://github.com/kubewarden/helm-charts/releases/). Alternatively the `imagelist.txt` and `policylist.txt` files are shipped inside the helm charts containing the used container images and policy wasm modules, respectively.
 
-```
-helm pull jetstack/cert-manager
+
+## Save artifacts in your workstation
+
+This command will download items described in artifacts.json into current directory. You can use `--dry` parameter to see what would happen if you executed the command.
+When the process completes, your current directory will contain downloaded archives. It will be present in the same directory where you executed the command.
+```bash
+# Print commands but do not execute
+airgap pull --list artifacts.json --dry
+
+# Start download
+airgap pull --list artifacts.json
 ```
 
+:::note
+Optionally, you can verify the signatures of the [helm charts](../../security/verifying-kubewarden#helm-charts) and [container images](../../security/verifying-kubewarden#container-images)
+:::
+
 ## Populate private registry
 
-Move `kubewarden-policies.tar.gz`, `kubewarden-images.tar.gz`, `kubewarden-load-images.sh`, `kubewarden-load-policies.sh` and `policies.txt`
-to the air gap environment.
+Now you upload archives from previous step into local registry. If you just want to play around you can start local registry with docker.
 
-1. Load Kubewarden images into the private registry. Docker client must be authenticated against the local registry
-```
-./kubewarden-load-images.sh --image-list ./kubewarden-images.txt --images kubewarden-images.tar.gz --registry <REGISTRY.YOURDOMAIN.COM:PORT>
-```
-2. Load Kubewarden policies into the private registry  
-```
-./kubewarden-load-policies.sh --policies kubewarden-policies.tar.gz --policies-list policies.txt --registry <REGISTRY.YOURDOMAIN.COM:PORT> --sources-path sources.yml 
+```bash
+# Start local repository (optional)
+# docker run -d -p 5000:5000 --restart=always --name registry registry:2
+
+# Push images and policies into local registry
+airgap push --list artifacts.json --registry=<REGISTRY.YOURDOMAIN.COM:PORT> --dry --insecure
 ```
 
-:::caution
 The `sources.yaml` file is needed by kwctl to connect to registries that fall into these categories:
-
 * Authentication is required
 * Self signed certificate is being used
 * No TLS termination is done
 
-Please refer to [the section on custom certificate authorities](../../distributing-policies/custom-certificate-authorities.md) in our documentation to learn more about configuring the `sources.yaml` file
-::: 
+You can use `--insecure` parameter to skip certificate check, but if you need more controll use [sources.yaml](../../distributing-policies/custom-certificate-authorities.md) file.
```

Comment:
```suggestion
You can use `--insecure` parameter to skip certificate check, but if you need more control use [sources.yaml](../../distributing-policies/custom-certificate-authorities.md) file.
```

Reply:
I think the note before this sentence on sources.yaml should be moved to follow this sentence stating that the user needs to use it along with kwctl in case they require finer control.